src/twisted/names/authority.py

```diff
@@ -302,17 +303,17 @@ class BindAuthority(FileAuthority):
     Supports only C{$ORIGIN} and C{$TTL} directives.
     """
 
-    def loadFile(self, filename):
+    def loadFile(self, filename: AnyStr) -> None:
         """
         Load records from C{filename}.
 
         @param filename: file to read from
-        @type filename: L{bytes}
+        @type filename: bytes or str C{AnyStr}
```

Comment:
This is not correct use of `AnyStr`.  `AnyStr` is a type variable used to ensure correspondence between bytes/str inputs and bytes/str outputs.  In a context like this, where an input can just be either but it doesn't affect anything else about the signature of the function, it isn't a  It definitely isn't a *third* type that could be returned. In any case, we just don't need `@type` annotations at all anymore, pydoctor supports annotations directly now.
```suggestion
```